src/components/NewTopHomeNextBar/NewTopHomeNextBar.js

```diff
@@ -8,16 +8,18 @@ import refresh from '../../assests/Images/refresh.svg';
 import { getLayout } from '../../utils/helper';
 
 const NewTopHomeNextBar = props => {
+  const lang_code = localStorage.getItem('apphomelang')
+      ? localStorage.getItem('apphomelang')
+      : 'en';
   const { trylink, nextlink, ishomeback, resultnextlang } = props;
-
   return (
     <div className="app_footbar_remove">
       <div className="row">
         {ishomeback ? (
           <>
             <div className="col s6 home_icon_div">
               <Link to={'/'}>
-                <img src={home} className="home_icon_new"></img>
+                <img src={home} className="home_icon_new" alt={lang_code === "en" ? "Home" : "வீடு"}></img>
```

Comment:
what about Hindi language?

Reply:
As discussed, we are using only English language text for now & will update them using language constant for respective language.